encord/objects/classification_instance.py

```diff
@@ -223,20 +239,11 @@ def set_for_frames(
         if last_edited_at is None:
             last_edited_at = datetime.now()
 
-        if self._range_only:
-            self._set_for_ranges(
-                frames=frames,
-                overwrite=overwrite,
-                created_at=created_at,
-                created_by=created_by,
-                confidence=confidence,
-                manual_annotation=manual_annotation,
-                last_edited_at=last_edited_at,
-                last_edited_by=last_edited_by,
-                reviews=reviews,
-            )
+        self._instance_data.update_from_optional_fields(
```

Comment:
Always set the instance data to the latest set, regardless if there's frame setting or not going on